node/cn/backend.go

```diff
@@ -320,8 +321,18 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 		}
 	}
 
+	var restartFn func()
```

Comment:
@kjhman21 This `restartFn` will be used if it is not nil as below code.
So It is ok. 

```
if self.restartFn != nil && self.restartTimeOut > 0 {
```

Reply:
Okay. I think it would be better to set an empty function to this variable and remove the condition in the `if`.
I don't mind anyway. It's your call. :)